README.md

```diff
@@ -68,9 +66,9 @@ On Windows, using [chocolatey](https://chocolatey.org):
 
     choco install chromedriver selenium-gecko-driver
 
-Finally, you can download manually [ChromeDriver](https://sites.google.com/a/chromium.org/chromedriver/) (for Chromium or Chrome)
-and [GeckoDriver](https://github.com/mozilla/geckodriver) (for Firefox) and put them anywhere in your `PATH`
-or in the `drivers/` directory of your project.
+Finally, you can manually download [ChromeDriver](https://sites.google.com/a/chromium.org/chromedriver/) (for Chromium or Chrome)
+or [GeckoDriver](https://github.com/mozilla/geckodriver) (for Firefox) and add them to your `PATH` or put them 
```

Comment:
`and` is better (you may need both).

Reply:
I changed it to "and/or", since not everybody needs both. OK?